connector/fyle_integrations_platform_connector/apis/base.py

```diff
@@ -36,30 +36,34 @@ def __get_last_synced_at(self):
         return ExpenseAttribute.get_last_synced_at(self.attribute_type, self.workspace_id)
 
 
-    def construct_query_params(self) -> dict:
+    def construct_query_params(self, sync_after: datetime = None) -> dict:
         """
         Constructs the query params for the API call.
         :return: dict
         """
-        
-        last_synced_record = self.__get_last_synced_at()
-        updated_at = self.format_date(last_synced_record.updated_at) if last_synced_record else None
+        if sync_after:
+            updated_at = self.format_date(sync_after)
+        else:
+            last_synced_record = self.__get_last_synced_at()
+            updated_at = self.format_date(last_synced_record.updated_at) if last_synced_record else None
 
         params = {'order': 'updated_at.desc'}
         params.update(self.query_params)
-        
-        if updated_at and self.attribute_type not in ('CATEGORY', 'EMPLOYEE', 'CORPORATE_CARD'):
+
+        if updated_at and self.attribute_type not in ('CATEGORY', 'EMPLOYEE', 'CORPORATE_CARD', 'MERCHANT'):
+            params['updated_at'] = updated_at
+        elif sync_after and updated_at:
```

Comment:
we should trust sync_after as a first priority. If it's present, always send it. If not, we can have additional conditions